From 1bb4e95d146227063fa9ab9a42e8eab347686d78 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:40:53 +0100
Subject: [PATCH 2/2] Squash of
 github/lineage-16.0(24216e9b8)..lineage-16.0-android-9.0.0_r31(68cbebc78) (DO
 NOT SUBMIT)

Bill Yi (2):
 - Import translations. DO NOT MERGE
 - Import translations. DO NOT MERGE

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (2):
 - Snap for 5044688 from 0869a7ceb54d98e90a7cacd953af93d4fb277422 to
   pi-qpr2-release
 - Snap for 5115288 from 7288a72ef623e03cff01f411529d7b7656dcef5f to
   pi-qpr2-release

Change-Id: I7066f29cd282c13b953e2d6d968923e895f8bca4

Change-Id: I61e45ce48a464f4f417f402d8d377c3bbc470a46
---
 res/values-bs/strings.xml | 6 +++---
 res/values-mr/strings.xml | 6 +++---
 res/values-my/strings.xml | 4 ++--
 res/values-nl/strings.xml | 2 +-
 4 files changed, 9 insertions(+), 9 deletions(-)

diff --git a/res/values-bs/strings.xml b/res/values-bs/strings.xml
index 2c8d29ba5..20fdf24f8 100644
--- a/res/values-bs/strings.xml
+++ b/res/values-bs/strings.xml
@@ -195,7 +195,7 @@
     <string name="accessibility_switch_to_camera" msgid="4518394037216725274">"Prebaci na sliku"</string>
     <string name="accessibility_switch_to_video" msgid="8174781871592793967">"Prebaci na snimanje videozapisa"</string>
     <string name="accessibility_switch_to_panorama" msgid="8322228859117808037">"Prebaci na panoramski način rada"</string>
-    <string name="accessibility_switch_to_photo_sphere" msgid="5803217570370854725">"Prebaci na način rada Foto-sfera"</string>
+    <string name="accessibility_switch_to_photo_sphere" msgid="5803217570370854725">"Prebaci na način rada za photo sphere fotografije"</string>
     <string name="accessibility_switch_to_gcam" msgid="7562625440767034695">"Prebaci na visoki kvalitet"</string>
     <string name="accessibility_switch_to_refocus" msgid="6796169367953860106">"Prebaci na način rada Promjena fokusa"</string>
     <string name="accessibility_review_cancel" msgid="5462850829869569629">"Prekini pregled"</string>
@@ -293,14 +293,14 @@
     <string name="mode_settings" msgid="2021937261522670921">"Postavke"</string>
     <string name="mode_camera" msgid="279763925715250603">"Kamera"</string>
     <string name="mode_video" msgid="8633929034048169139">"Videozapis"</string>
-    <string name="mode_photosphere" msgid="5082338476237291833">"Foto-sfera"</string>
+    <string name="mode_photosphere" msgid="5082338476237291833">"Photo sphere fotografija"</string>
     <string name="mode_timelapse" msgid="1517168724627815453">"Protok vremena"</string>
     <string name="mode_wideangle" msgid="1099640345041696830">"Široki ugao"</string>
     <string name="mode_panorama" msgid="6882648621305828861">"Panoramski način rada"</string>
     <string name="mode_refocus" msgid="172192967119469061">"Efekt s izoštr. objektom"</string>
     <string name="mode_camera_desc" msgid="3670807361165657278">"Prebaci na način rada za kameru"</string>
     <string name="mode_video_desc" msgid="4058407412929464689">"Prebaciti na videokameru"</string>
-    <string name="mode_photosphere_desc" msgid="2403816283825579161">"Prebaci na način rada Foto-sfera"</string>
+    <string name="mode_photosphere_desc" msgid="2403816283825579161">"Prebaci na način rada za photo sphere fotografije"</string>
     <string name="mode_panorama_desc" msgid="6023020404237519590">"Prebaci na panoramski način rada"</string>
     <string name="mode_refocus_desc" msgid="6171183297515508465">"Prebaci na način rada Efekt s izoštrenim objektom"</string>
     <string name="settings_open_desc" msgid="6914471293841793766">"Otvori postavke"</string>
diff --git a/res/values-mr/strings.xml b/res/values-mr/strings.xml
index 5e63397ed..7dc363e4a 100644
--- a/res/values-mr/strings.xml
+++ b/res/values-mr/strings.xml
@@ -68,7 +68,7 @@
     <string name="error_cannot_connect_camera" msgid="2713059773224193128">"कॅमेर्‍याशी कनेक्ट करु शकत नाही."</string>
     <string name="error_camera_disabled" msgid="5582398275657419692">"सुरक्षा धोरणांमुळे कॅमेरा अक्षम केला गेला आहे."</string>
     <string name="error_media_storage_failure" msgid="7711687330673245629">"तुमचा फोटो किंवा व्हिडिओ सेव्ह करण्यात समस्या आली."</string>
-    <string name="error_permissions" msgid="3454167403425651254">"अॅपला चालविण्यासाठी आवश्यक गंभीर परवानग्या नाहीत. कृपया आपल्या परवानग्या सेटिंग्ज तपासा."</string>
+    <string name="error_permissions" msgid="3454167403425651254">"अॅपला चालविण्यासाठी आवश्यक गंभीर परवानग्या नाहीत. कृपया तुमच्या परवानग्या सेटिंग्ज तपासा."</string>
     <string name="reason_storage_failure" msgid="6818898304774654969">"फोटो संचयन बिघाड."</string>
     <string name="wait" msgid="765601745709933047">"कृपया प्रतीक्षा करा..."</string>
     <string name="no_storage" product="default" msgid="5922012950780548098">"कॅमेरा वापरण्‍यापूर्वी SD कार्ड घाला."</string>
@@ -160,7 +160,7 @@
     <string name="dialog_cancel" msgid="692365061128351656">"रद्द करा"</string>
     <string name="dialog_report" msgid="7616428760369876209">"अहवाल द्या"</string>
     <string name="dialog_dismiss" msgid="1382550604990470119">"डिसमिस करा"</string>
-    <string name="spaceIsLow_content" product="default" msgid="4522771065344332702">"आपल्या SD कार्डावरील स्थान समाप्त झाले आहे. गुणवत्ता सेटिंग बदला किंवा काही इमेज किंवा इतर फायली हटवा."</string>
+    <string name="spaceIsLow_content" product="default" msgid="4522771065344332702">"तुमच्या SD कार्डावरील स्थान समाप्त झाले आहे. गुणवत्ता सेटिंग बदला किंवा काही इमेज किंवा इतर फायली हटवा."</string>
     <string name="video_reach_size_limit" msgid="9196836111505731836">"आकार मर्यादा गाठली."</string>
     <string name="pano_too_fast_prompt" msgid="2503148095578052177">"खूप जलद"</string>
     <string name="pano_dialog_prepare_preview" msgid="4445235163599534263">"पॅनोरामा तयार करत आहे"</string>
@@ -369,7 +369,7 @@
     <string name="pref_camera_exposure_compensation" msgid="4143245817259719147">"व्‍यक्तिचलित प्रदर्शन"</string>
     <string name="settings_cling_text" msgid="5368054627268181292">"सेटिंग्ज मध्ये, रिझॉल्यूशन आणि गुणवत्ता बदला किंवा प्रगत वैशिष्ट्ये वापरून पहा."</string>
     <string name="photo_size_selection_title" msgid="2789753590719172645">"फोटो आकार निवडा"</string>
-    <string name="photo_aspect_ratio_selection_content" msgid="6668224437962196229">"आपण हे नंतर सेटिंग्ज मध्ये बदलू शकता."</string>
+    <string name="photo_aspect_ratio_selection_content" msgid="6668224437962196229">"तुम्ही हे नंतर सेटिंग्ज मध्ये बदलू शकता."</string>
     <string name="share_to" msgid="5144911209144798122">"यावर शेअर करा"</string>
     <string name="edit_with" msgid="615569327230783971">"सह संपादित करा"</string>
     <string name="startup_dialog_button_next" msgid="1011831256978228993">"पुढील"</string>
diff --git a/res/values-my/strings.xml b/res/values-my/strings.xml
index 7f6524bb8..9406b2be3 100644
--- a/res/values-my/strings.xml
+++ b/res/values-my/strings.xml
@@ -374,8 +374,8 @@
     <string name="edit_with" msgid="615569327230783971">"...နှင့် တည်းဖြတ်ရန်"</string>
     <string name="startup_dialog_button_next" msgid="1011831256978228993">"ရှေ့သို့"</string>
     <string name="confirm_button_text" msgid="7389949384482206814">"OK၊ ရပါပြီ"</string>
-    <string name="full_sensor_4x3_aspect_ratio" msgid="1270461419743888925">"အာရုံခံပုံ အပြည့် \n(4:3)"</string>
-    <string name="cropped_sensor_16x9_aspect_ratio" msgid="4742161537633251795">"အာရုံခံကိရိယာမှ ဖြတ်ထား\n(16:9)"</string>
+    <string name="full_sensor_4x3_aspect_ratio" msgid="1270461419743888925">"အာရုံခံပုံ အပြည့် \n(၄x၃)"</string>
+    <string name="cropped_sensor_16x9_aspect_ratio" msgid="4742161537633251795">"အာရုံခံကိရိယာမှ ဖြတ်ထား\n(၁၆x၉)"</string>
     <string name="pref_boolean_false" msgid="461317129020087333">"၀"</string>
     <string name="pref_boolean_true" msgid="4131945223998878009">"၁"</string>
     <string name="session_saving_image" msgid="8932315404486416826">"သိမ်းနေစဉ်…"</string>
diff --git a/res/values-nl/strings.xml b/res/values-nl/strings.xml
index 9ed674ec5..b12090c63 100644
--- a/res/values-nl/strings.xml
+++ b/res/values-nl/strings.xml
@@ -346,7 +346,7 @@
     <string name="setting_front_camera_photo" msgid="4131886734622868637">"Fotocamera voorzijde"</string>
     <string name="setting_front_camera_video" msgid="2178799452805359752">"Videocamera voorzijde"</string>
     <string name="setting_default_camera" msgid="6954076799301004779">"Standaardcamera"</string>
-    <string name="setting_google_help_and_feedback" msgid="2079580537079242775">"Help en feedback"</string>
+    <string name="setting_google_help_and_feedback" msgid="2079580537079242775">"Hulp en feedback"</string>
     <string name="processing_hdr_plus" msgid="9160093263037540304">"HDR+ verwerken…"</string>
     <string name="open_source_licenses" msgid="2169711954264883060">"Open-sourcelicenties"</string>
     <string name="pref_category_general" msgid="6737748849700581019">"Algemene instellingen"</string>
-- 
2.17.1

